backend/tests/unit/models/test_websocket_models.py

```
@@ -55,33 +56,25 @@
+    ])
+    def test_subscription_message_creation(self, message_data, expected_type, expected_channel):
+        """Test subscription message creation with various types."""
+        message = SubscriptionMessage(**message_data)
+        assert message.type == expected_type
+        assert message.channel == expected_channel
+
+    @pytest.mark.parametrize("invalid_data,expected_error", [
+        ({"type": "invalid", "channel": "test"}, "Invalid subscription type"),
+        ({"type": "subscribe"}, "Missing required field"),
+    ])
+    def test_subscription_message_validation(self, invalid_data, expected_error):
```

Ruff (0.12.2)

Unused method argument: `expected_error`

(ARG002)